src/CMakeLists.txt

```diff
@@ -102,3 +102,16 @@ set_target_properties(hotspot
 
 install(TARGETS hotspot RUNTIME DESTINATION ${KDE_INSTALL_BINDIR})
 install(FILES hotspot.notifyrc DESTINATION ${KDE_INSTALL_KNOTIFY5RCDIR})
+
+if (KF5Auth_FOUND)
+    target_link_libraries(hotspot KF5::AuthCore)
+    target_compile_definitions(hotspot PRIVATE USE_KAUTH)
```

Comment:
use `hotspot-config.h.cmake` for this purpose with a new `#cmakedefine01` therein